dataactcore/scripts/initialize.py

```diff
@@ -142,6 +146,8 @@ def main():
     parser.add_argument('-z', '--load_zips', help='Load zip code data', action='store_true')
     parser.add_argument('-o', '--load_offices', help='Load FPDS Office Codes', action='store_true')
     parser.add_argument('-u', '--uncache_all_files', help='Un-cache file generation requests', action='store_true')
+    parser.add_argument('-f', '--force', help='Forces actions to occur in certain scripts regardless of checks',
```

Comment:
Last time we talked about this we decided to leave the shorthand ones for dev use and that they weren't hurting anything by existing. Is this no longer the case?

Also, I'm not sure we want a single one different from the rest. For now we probably at least want to be consistent with them.

Reply:
While I like having both short and long arguments for general scripts, `initialize.py` may be an exception given how there are so many arguments to it that the shorthand versions lose their meaning. That being said, this can be held off if/when we decouple and rework `initialize.py`.